RecoHGCal/TICL/python/HADStep_cff.py

```diff
@@ -38,32 +38,37 @@
     ,filteredLayerClustersHAD
     ,ticlTrackstersHAD)
 
-filteredLayerClustersHFNoseHAD = _filteredLayerClustersProducer.clone(
+# HFNOSE CLUSTER FILTERING/MASKING
+
+filteredLayerClustersHFNoseHAD = filteredLayerClustersHAD.clone(
     min_cluster_size = 2, # inclusive
     algo_number = 9,
     iteration_label = "HADn",
     LayerClusters = 'hgcalLayerClustersHFNose',
-    LayerClustersInputMask = "hgcalLayerClustersHFNose:InitialLayerClustersMask"
+    LayerClustersInputMask = "ticlTrackstersHFNoseTrk"
 )
 
-ticlTrackstersHFNoseHAD = _trackstersProducer.clone(
+# HFNOSE CA - PATTERN RECOGNITION
+
+ticlTrackstersHFNoseHAD = ticlTrackstersHAD.clone(
     detector = "HFNose",
     layer_clusters = "hgcalLayerClustersHFNose",
     layer_clusters_hfnose_tiles = "ticlLayerTileHFNose",
-    original_mask = "hgcalLayerClustersHFNose:InitialLayerClustersMask",
+    original_mask = "ticlTrackstersHFNoseTrk", #"ticlTrackstersHFNoseTrk",
```

Comment:
what is the value of `#"ticlTrackstersHFNoseTrk",` here? it's not suggesting an alternative and is apparently superfluous. Please check and cleanup or clarify why this part is needed.

Reply:
Didn't notice that